cmd/signozschemamigrator/schema_migrator/traces_migrations.go

```diff
@@ -1,3 +1,360 @@
 package schemamigrator
 
 var TracesMigrations = []SchemaMigrationRecord{}
+
+// move them to TracesMigrations once it's ready to deploy
+var TracesMigrations_Staging = []SchemaMigrationRecord{
+	{
+		MigrationID: 1000,
+		UpItems: []Operation{
+			CreateTableOperation{
+				Database: "signoz_traces",
+				Table:    "signoz_index_v3",
+				Columns: []Column{
+					{Name: "ts_bucket_start", Type: ColumnTypeUInt64, Codec: "DoubleDelta, LZ4"},
+					{Name: "resource_fingerprint", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "timestamp", Type: DateTime64ColumnType{Precision: 9}, Codec: "DoubleDelta, LZ4"},
+					{Name: "id", Type: FixedStringColumnType{Length: 27}, Codec: "ZSTD"},
+					{Name: "traceID", Type: FixedStringColumnType{Length: 32}, Codec: "ZSTD(1)"},
+					{Name: "spanID", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "traceState", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "parentSpanID", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "flags", Type: ColumnTypeUInt32, Codec: "T64, ZSTD(1)"},
+					{Name: "name", Type: LowCardinalityColumnType{ColumnTypeString}, Codec: "ZSTD(1)"},
+					{Name: "kind", Type: ColumnTypeInt8, Codec: "T64, ZSTD(1)"},
+					{Name: "spanKind", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "durationNano", Type: ColumnTypeUInt64, Codec: "T64, ZSTD(1)"},
+					{Name: "statusCode", Type: ColumnTypeInt16, Codec: "T64, ZSTD(1)"},
+					{Name: "statusMessage", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "statusCodeString", Type: ColumnTypeString, Codec: "ZSTD(1)"},
```

Comment:
Why continue the inconsistent snake case for new tables?

Reply:
We wanted to keep them as it as and have alias to suppor the new enrichment queries.

Reply:
Do you mean to revert i.e have new name and old alias ?

Reply:
Yes, new name and an old alias with a column comment saying deprecated.

Reply:
I didn't do that because I was unsure about the composite columns(attributes), so I kept all of them as they were and added aliases for the ones that can be supported.

Are you suggesting to keep composite ones as it is ?

Reply:
My point is why not clean up the inconsistency and mess from v2? Like change this from

```
					{Name: "statusCode", Type: ColumnTypeInt16, Codec: "T64, ZSTD(1)"},
					{Name: "statusMessage", Type: ColumnTypeString, Codec: "ZSTD(1)"},
					{Name: "statusCodeString", Type: ColumnTypeString, Codec: "ZSTD(1)"},
```
to 

```
					{Name: "status_code", Type: ColumnTypeString, Codec: "ZSTD(1)"},
					{Name: "status_message", Type: ColumnTypeString, Codec: "ZSTD(1)"},

```

And do the same for all.

Reply:
ok

Reply:
Have done the changes, 

However I would like some thoughts on `custom composite columns`. If we should keep their name as it is. Since for the simple attribute columns I was able to convert them to a easier format.

Reply:
I see you created snake case column and used them in aliasing. I don't see issue with that.